packages/kbn-reporting/generate_csv/types/index.d.ts

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ByteSizeValue } from '@kbn/config-schema';
+import type { IUiSettingsClient, Logger } from '@kbn/core/server';
+import { createEscapeValue } from '@kbn/data-plugin/common';
+
+const { PDF_JOB_TYPE, PDF_JOB_TYPE_V2, PNG_JOB_TYPE, PNG_JOB_TYPE_V2 } = jobTypes;
+
+export const PLUGIN_ID = 'reporting';
+
+export const REPORTING_TRANSACTION_TYPE = PLUGIN_ID;
+
+export const REPORTING_SYSTEM_INDEX = '.reporting';
+
+export const JOB_COMPLETION_NOTIFICATIONS_SESSION_KEY =
+  'xpack.reporting.jobCompletionNotifications';
+
+export const CONTENT_TYPE_CSV = 'text/csv';
+export const CSV_REPORTING_ACTION = 'downloadCsvReport';
+export const CSV_BOM_CHARS = '\ufeff';
+export const CSV_FORMULA_CHARS = ['=', '+', '-', '@'];
```

Comment:
The constants that are needed from the package context should probably go in the new `@kbn/reporting/common`

Reply:
[95e879a](https://github.com/elastic/kibana/pull/151801/commits/95e879a32b974d188dc7840ccc68af9910543094) 💥